config/settings/test.yml

```diff
@@ -468,3 +472,7 @@ travel_pay:
 
 pension_ipf_vanotify_status_callback:
   bearer_token: bearer_token_secret
+
+va_notify:
+  status_callback:
+    bearer_token: 'va_notify_bearer_token'
```

Comment:
Did you mean to add this `va_notify` section twice in this file?

Reply:
thanks for catching this.